### ISSUE TITLE
Work with commented code?

### ISSUE THREAD

**Issue by PixelT** [CoenraadS/BracketPair#162]
It would be great if this extension can also work & match brackets on commented code too.

**Comment by jonathan-irvin** [comment]
I disagree...to a point.  Commented code is tech-debt.  The editor ignores it, and so do developers.

If you're using a decent editor, it should already color your commented code for you.  I can somewhat agree with matching block comments like ["/*,"*/"] and [""], but again, what purpose does it serve?

I would remove commented code altogether and make a git commit for it and if you need it in the future, revert that commit.
